src/cli/utils.ts

```diff
@@ -302,7 +302,8 @@ export function paginationParamsFromOptions<O extends PaginationCLIOptions>(
   return {
     cursor,
     limit: numberLimit,
-    sortBy,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    sortBy: sortBy as any,
```

Comment:
nit: does `unknown` work better here? otherwise you can provide another type to the generic and do a SortBy<T> on it.

Reply:
got another generic here for types c29bc32

we're not yet asserting the types though. would need a constant for each set of valid sort bys or derive them from an instantiated object of each